thoth/storages/buildlogs_analyses_cache.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# thoth-storages
+# Copyright(C) 2019 Harshad Reddy Nalla
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Adapter for retrieving and storing hashes for fast buildlog lookups."""
+
+from .ceph_cache import CephCache
+
+
+class BuildLogsAnalysesCacheStore(CephCache):
```

Comment:
When this cache will be used? What is the use case here?

Reply:
I have added it, so that when build-analyzer is scheduled we store the result in cache as well and check in with the cache before scheduling.  Still a work in progress.

Reply:
Nice! Good idea. :+1: